src/app/pages/forgot-password/update-password/update-password.component.ts

```diff
@@ -46,7 +46,7 @@ export class UpdatePasswordComponent implements OnInit, OnDestroy {
         takeUntil(this.destroy$)
       )
       .subscribe(() => {
-        this.router.navigate(['/login']);
+        this.router.navigate(['/login'], { queryParams: { forcePageView: 'true' } });
```

Comment:
I would think a string is not the better choice here. We should just use as simple boolean instead, as is done at other places as well.
This will make the condition in the guard nicer to read too.